### DISCUSSION TITLE
Control of Portainer Stacks created outside of Portainer

### DISCUSSION THREAD

**Discussion by tomhusband** [tteck/Proxmox#2220] [Q&A]
I used the tteck script to install an LXC container in Proxmox. Along with it came the option to install Docker, Portainer and Compose. In the container I setup Frigate , a network video recorder. Now in Portainer I'd like to get to `docker-compose.yml` so I can remove `:ro` from the file in order to have write capability in the Frigate GUI config file.
How do I get access to Compose in Portainer? If I go to Stacks I see access to the `.yml` file is limited because it was created outside of Portainer. How do I change that?

Maybe this isn't the right place for the question. If not I'll remove it.

**Comment by tteck** [comment]
Sounds like you should have chosen Dockge.
Dockge is a versatile tool that simplifies Docker management, especially when working with Docker Compose.yaml files.

**Reply by tomhusband** [reply]
Hmmm, so I guess installing Dockage now isn't going to help?

**Reply by tteck** [reply]
I recommend creating a Dockge LXC and pasting your `compose.yaml` file into Dockge. This test will help determine if it meets your requirements. Before conducting the test, make sure to shutdown your current Docker LXC.

**Reply by tomhusband** [reply]
OK thanks. Maybe I'll try that after I'm more familiar with everything. It's all real new to me.

**Reply by tteck** [reply]
A Dockge LXC is just a copy/paste and enter away. 😉
